packages/linear-genome-view/src/BasicTrack/components/BlockBasedTrack.js

```diff
@@ -1,17 +1,24 @@
+/* eslint-disable no-nested-ternary */
 import { getConf } from '@gmod/jbrowse-core/configuration'
 import { observer, PropTypes as MobxPropTypes } from 'mobx-react'
 import { getParent } from 'mobx-state-tree'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Track from './Track'
 import TrackBlocks from './TrackBlocks'
+import AlignmentsTrackBlocks from './AlignmentsTrackBlocks'
 
 function BlockBasedTrack(props) {
   const { model, children } = props
   return (
     <Track {...props} trackId={getConf(model, 'trackId')}>
       {model.trackMessageComponent ? (
         <model.trackMessageComponent model={model} />
+      ) : getConf(model, 'type') === 'AlignmentsTrack' ? (
```

Comment:
Can't detect AlignmentsTrack here, need to find another way of doing this.  Probably just make a special react component for this track.

Reply:
special react component made inside the AlignmentsTrack -> component folder. Now AlignmentsTrack returns <AlignmentsBlockBasedTrack/> rather than the BasicTrack's <BlockBasedTrack/>